docs/source/resources/scyllaclusters/multidc/multidc.md

```diff
@@ -107,8 +107,8 @@ metadata:
   name: scylla-cluster
   namespace: scylla
 spec:
-  agentVersion: 3.4.0
-  version: 6.2.0
+  agentVersion: {{agentVersion}}
```

Comment:
It doesn't render correctly - try it locally.

![image](https://github.com/user-attachments/assets/2ab44a34-0682-450b-94ef-775069f9d336)


I recall there was a related issue but the best pointer I can find now is this: https://github.com/scylladb/sphinx-scylladb-theme/issues/1290.

Reply:
After offline discussion: excluded from this PR. We're no longer changing the version literals to variable expansion.